src/eva/core/data/datasets/embeddings/classification/multi_embeddings.py

```diff
@@ -84,6 +88,12 @@ def _load_embeddings(self, index: int) -> torch.Tensor:
         if not embeddings.ndim == 2:
             raise ValueError(f"Expected 2D tensor, got {embeddings.ndim} for {multi_id}.")
 
+        if embeddings.shape[0] < self._n_embeddings:
```

Comment:
The padding logic is not necessary here, we can do that via the `embedding_transforms`.
See example here:
https://github.com/kaiko-ai/eva/blob/767fbbd399961560e0796e4c31187225584c791e/tests/eva/core/data/datasets/embeddings/classification/test_multi_embeddings.py#L88